exercises/concept/lucas-numbers/.meta/exemplar.ex

```diff
@@ -0,0 +1,31 @@
+defmodule LucasNumbers do
+  @moduledoc """
+  Lucas numbers are an infinite sequence of numbers which build progressively
+  which hold a strong correlation to the golden ratio (φ or ϕ)
+
+  E.g.: 2, 1, 3, 4, 7, 11, 18, 29, ...
+  """
+  @spec generate(integer()) :: list(integer())
+  def generate(count) when not is_integer(count) or count < 1 do
+    raise ArgumentError, "count must be specified as an integer >= 1"
+  end
+
+  def generate(1) do
+    [2]
+  end
+
+  def generate(2) do
+    [2, 1]
+  end
```

Comment:
I'm not sure this is needed at all. If I remove those cases, all the tests still pass. most likely because `[2 | []]` === `[2]`.

Reply:
Agreed, I don't think they are necessarily needed either, but I didn't want someone to get tripped over the mathiness of this exercise